HonorAssistChatMessage.lua

```diff
@@ -39,7 +39,7 @@ function HonorAssist:CreateHonorableKillMessage(estimatedHonorGained, playerKill
 
 	text = HonorAssist:GetTranslation("YOU_HAVE_KILLED") .. ' ' .. playerKilled .. ' (' .. playerRank .. ') ' .. timesKilled .. ' ' .. HonorAssist:GetTranslation(timeText) .. '. '
 		.. HonorAssist:GetTranslation("THIS_KILL_GRANTED") .. ' ' .. percentage * 100 .. '% ' .. HonorAssist:GetTranslation("VALUE_FOR") .. ' |cFF00ccff' .. realisticHonor .. ' ' 
-		.. HonorAssist:GetTranslation("HONOR"):lower() .. '|cFF' .. HonorAssist:RGBPercToHex(chatInfo.r, chatInfo.g, chatInfo.b) .. ' ' .. string.match(text, "(%(.+)") .. '.'
```

Comment:
So removing this causes issues in other parts of the UI (will be more noticeable when history feature goes in). We use both "Honor" and "honor" in areas which is why we lowercase it dynamically. I don't think we should be making this change as a result.

Reply:
If we use the term "Honor" in titles and so on, it should be a different localized variable than "honor". Because in the german translation "Ehre":lower() results in "ehre" which is grammatical wrong. Such things like lower() should not be done on every translation. But I will make some changes to that part of code this afternoon, since I really dislike the concatination of strings to build the result. That limits the translations, because the order is fixed and can't be changed. Gonna try to use string.format(s, e1, e2, ...)

Reply:
Ah, okay. If that's the case then we'll have to create some more translations. I'm all for changing how the string is generated. I think temporarily we'll want a check to use that for the German client only because removing the :lower() for some regions will be weird, and for some it's already weird. I would rather leave the regions that are correct in their current state and the ones that are already weird can stay weird until a pass is made to update their translations.

Reply:
I guess I still dont understand the reason for the :lower() .... if you want it to be lowercase for english version, you can simply write it in lower letters in the enEN.lua?

Reply:
You're right. The problem is that it is not currently that way in enEN.lua (or any other regions). That means removing the :lower() call negatively impacts US region and has unknown impact on others while improving Germany. That's why initially there needs to be an if-statement that has Germany using the new string concatenation you wanted to implement, and leave all the other regions using this same one until someone can go through and verify the translations.

Reply:
You could also remove the :lower and add :upper to the other places where the string is used instead.

Reply:
You can't add :upper() either. The reason it's the way it is now is because it was assumed (incorrectly) that we could create one single word for Honor and use it interchangeably in situations where we need to use Honor or honor by caller :lower(). We don't use "HONOR" anywhere.

Correct, distinct strings would be the ideal way. The problem is as this pull request currently stands you fix a problem for deDE clients but negatively impact others. I'll make more official changes here once I get time (Holidays will slow stuff down). 

Flip flopping who is negatively impacted isn't the answer. The solution I suggested above would fix the problem for deDE clients and leave the rest unchanged until we can get around to fixing translations for other clients. Otherwise you'll need to wait for me to come around to fixing the translations for all clients before the :lower() call can be removed.